pkg/minikube/update/update.go

```diff
@@ -49,64 +50,86 @@ const (
 	githubRepo  = "minishift"
 )
 
-var (
-	lastUpdateCheckFilePath = constants.MakeMiniPath("last_update_check")
-)
-
-func MaybeUpdateFromGithub(output io.Writer) {
+func MaybeUpdateFromGithub(output io.Writer) error {
 	localVersion, err := version.GetSemverVersion()
 	if err != nil {
 		glog.Errorln(err)
```

Comment:
let's remove the  log and be consistent. once you log once you don't. Let's just return the error and let the caller deal with it